python/ambassador/envoy/v3/v3listener.py

```diff
@@ -447,18 +447,21 @@ def base_http_config(self) -> Dict[str, Any]:
             base_http_config["tracing"] = {}
             self.traffic_direction = "OUTBOUND"
 
+            finalCustomTags = []
+            custom_tags = self.config.ir.tracing.get('custom_tags', [])
             req_hdrs = self.config.ir.tracing.get('tag_headers', [])
-
-            if req_hdrs:
-                base_http_config["tracing"]["custom_tags"] = []
-                for hdr in req_hdrs:
-                    custom_tag = {
-                        "request_header": {
-                            "name": hdr,
+            if custom_tags or req_hdrs:
+                finalCustomTags += custom_tags
+                if req_hdrs:
```

Comment:
I thought we already translated these in the CRD conversion work from `v2` to `v3alpha1`? 

AFAIK, by the time we get here the values have been converted to `v3alpha1`. Do we need to still translate or can we assume at this point that the `custom_tags` has everything we want already?

Reply:
That's probably true. I have simplified it now. Will need to properly test that. Running tests locally is taking ages.